packages/react-core/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -18,15 +19,15 @@ export enum ToolbarItemVariant {
 export interface ToolbarItemProps extends React.HTMLProps<HTMLDivElement> {
   /** Classes applied to root element of the data toolbar item */
   className?: string;
-  /** TODO: Support 'separator' element as a <Divider component="div" variant="horizontal" />. A type modifier which modifies spacing specifically depending on the type of item */
```

Comment:
Add back comment for docs: `/** A type modifier which modifies spacing specifically depending on the type of item */`